### PR TITLE
Feature/j query

### PR DESCRIPTION
**Pull request by CLPena**
### Type of change made:
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Where should the reviewer start?
Review scripts and domUpdates files
### How should this be manually tested?
Test functionality
### Any background context you want to provide?
N/A
### What are the relevant tickets?
N/A
### Screenshots (if appropriate)
N/A
### Questions:
N/A
#### src directory
  - [ ] base.scss
  - [ ] styles.scss
  - [ ] cookbook.js
  - [ ] pantry.js
  - [ ] recipe.js
  - [ ] user.js
  - [x] scripts.js
- [x] domUpdates.js
  - [ ] index.html
#### test directory
- [ ] cookbook-test.js
- [ ] pantry-test.js
- [ ] recipe-test.js
- [ ] user-test.js

### CODE REVIEW

**Comment by TimNguyen21** [comment]
Converting from scripts to jQuery was a challenge, but we got the functionality to work.

**Comment by J-Poulter** [comment]
We worked on this together

File: src/domUpdates.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+      $('.ingredients-confirmation').text(`You have all the ingredients needed for this recipe!`);
+      $('.ingredients-cost').text('');
+    }
+    $(".buy-ingredients").click(function() {
+      pantry.updatePantryContent(user, recipeObject);
+    })
+
+    $(".cook-recipe").click(function() {
+      pantry.removeConsumedIngredients(user, recipeObject);
+    })
```
Comment (TimNguyen21) [inline src/domUpdates.js]: We can separate this to a helper function that passing the arguments of user and recipeObject.

File: src/domUpdates.js [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+  },
+
+  displayDirections(event, cookbook, pantry, ingredientData, recipeData, user) {
+
+    let newRecipeInfo = cookbook.recipes.find(recipe => {
+      if (recipe.id === Number(event.target.dataset.id)) {
+        return recipe;
+      }
+    })
+    recipeObject = new Recipe(newRecipeInfo, ingredientData, recipeData);
```
Comment (TimNguyen21) [inline src/domUpdates.js]: We can rename this since we don't want to name variables with the key name of Object

**Pull request merged**